skins/cat17/templates/partials/payment_intervals.html.twig

```diff
@@ -21,7 +21,7 @@
 		</div>
 
 		<p class="info-text" data-info="{$ PAYMENT_INTERVAL_OPTIONS[interval].id $}">
-			{$ ( 'payment_interval_' ~ interval ~ '_info' ) | trans $}
+			{$ ( context ~ '_payment_interval_' ~ interval ~ '_info' ) | trans $}
```

Comment:
We don't follow a rule set (don't know a formal one), but would you mind adding the context parameter to the documentation in the head of `payment_intervals.html.twig` and maybe two examples to prevent us from making mistakes / going mad looking for those keys (from here to content repo or the other way around), now that they are so very dynamic...